lms/models/lti_user.py

```diff
@@ -89,7 +89,11 @@ def is_learner(self):
     @property
     def is_admin(self):
         """Whether this user is an admin."""
-        return any(role.type == RoleType.ADMIN for role in self.lti_roles)
+        return any(
+            role.type == RoleType.ADMIN
+            and role.scope in [RoleScope.COURSE, RoleScope.SYSTEM]
```

Comment:
Introducing the scope filter here.

Scopes are "course" (self explanatory), system (system-wide) and institution (which means anywhere in this instutiton, but not here)